keras_nlp/api/models/__init__.py

```diff
@@ -74,6 +75,9 @@
 from keras_nlp.src.models.deberta_v3.deberta_v3_tokenizer import (
     DebertaV3Tokenizer,
 )
+from keras_nlp.src.models.deeplab_v3_plus.deeplab_v3_plus_segmenter import (
```

Comment:
General comment, if we think that we can handle deep lab v3 and deep lab v3+ from the same class, can we rename this whole directory and all symbols to just `DeepLabV3`? The shorter name would be nice. "plus" can exist only in the preset name.

Reply:
Changed